frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test

```diff
@@ -0,0 +1,150 @@
+Feature: Topology smoke test
+	As a user, I want see a page for Pipeline
+
+Background:
+    Given user logged into the openshift application
+    
+
+@regression, @smoke
+Scenario: Empty state of topology
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user selects a existing project from project list or create a new project
+   Then user sees Add page with message on the top "No workloads found"
+
+@regression, @smoke
+Scenario: Topology with workloads
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user selects a existing project from project list with existing workloads
+   Then user sees different workloads in topology chart area
+
+@regression, @smoke
+Scenario: Visual for deployment/deployment-config
+   Given user is in topology with deployment and deployment-config workload
+   When user checks nodes and the decorators associated with them
+   And nodes are circular shaped with builder image in them
+   And pod ring associated with node are present around node with color according to the pod status
+   And deployment and deployment-config can have application url on top-right of the node 
+   And user sees edit source code decorator is on bottom right of the node which can lead to github or che workspace
+   And user sees build decorator on bottom left which will take user to either build tab or pipeline depending on pipeline associated with them 
+   Then user checks node label having "D" for deployment and "DC" for deployment-config and then name of node
+
+@regression, @smoke
+Scenario: Visual for knative service
```

Comment:
split this scenario into multiple scenarios for each item 
Eg: 
1. knaitve workload with no revisions
2. components of the knative revision in topology page
3. traffic distribution details of kantive revision display etc..